### PR TITLE
Add bios boot integration cases

### PR DESCRIPTION
**Pull request by hahakiki2010**
Boot Uefi/Seabios guest, and test
vm undefine/define/start/destroy/save/restore
operations

Signed-off-by: Yan Li <yannli@redhat.com>

### CODE REVIEW

**Comment by hahakiki2010** [comment]
This PR depends on #https://github.com/avocado-framework/avocado-vt/pull/1825

**Comment by hahakiki2010** [comment]
Test passed on q35 by ovmf
```
(.libvirt-ci-venv-ci-runtest-F8mimU) [root@lenovo-sr630-02 tp-libvirt]# avocado run --vt-type libvirt boot_integration.ma_i440fx.by_seabios
JOB ID     : b96fa29f7c9530adef0573fdec95f4cbf6b8b5d6
JOB LOG    : /root/avocado/job-results/job-2018-11-16T01.19-b96fa29/job.log
 (1/2) type_specific.io-github-autotest-libvirt.boot_integration.ma_i440fx.by_seabios.boot_dev.define_start_destroy_save_restore_undefine: -

PASS (67.33 s)
 (2/2) type_specific.io-github-autotest-libvirt.boot_integration.ma_i440fx.by_seabios.boot_order.define_start_destroy_save_restore_undefine: PASS (63.39 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 144.16 s
```
Test passed on i440fx by seabios
```
(.libvirt-ci-venv-ci-runtest-6KzwH0) [root@hp-dl320eg8-05 bios]# avocado run --vt-type libvirt boot_integration.ma_q35.by_ovmf
JOB ID     : 9ffb677b9b30d4e74240a7b99b57cde70194e35a
JOB LOG    : /root/avocado/job-results/job-2018-11-16T01.22-9ffb677/job.log
 (1/2) type_specific.io-github-autotest-libvirt.boot_integration.ma_q35.by_ovmf.boot_dev.define_start_destroy_save_restore_undefine: PASS (54.14 s)
 (2/2) type_specific.io-github-autotest-libvirt.boot_integration.ma_q35.by_ovmf.boot_order.define_start_destroy_save_restore_undefine: PASS (56.02 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 125.39 s
```

File: libvirt/tests/src/bios/boot_integration.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    if (boot_type == "seabios" and
+            not utils_package.package_install('seabios-bin')):
+        test.error("seabios package install failed")
+
+    try:
+        prepare_boot_xml(vmxml, params)
+
+        # Update domain disk
+        # Use sata for uefi, and ide for seabios
+        domain_disk = vmxml.get_devices(device_type='disk')[0]
```
Comment (chunfuwen) [inline libvirt/tests/src/bios/boot_integration.py]: I assume line 100 to105 can be achieved by libvirt.set_vm_disk(vm, params)

Comment (hahakiki2010) [reply libvirt/tests/src/bios/boot_integration.py]: I just want to update the disk target info, and keeps others. Use set_vm_disk is too complex

**Pull request merged**